classes/managers/NostoAdminTabManager.php

```diff
@@ -31,6 +31,7 @@ class NostoAdminTabManager
 
     const MAIN_MENU_ITEM_CLASS = 'AdminNosto';
     const SUB_MENU_ITEM_CLASS = 'AdminNostoPersonalization';
+    const ID_LANG = "id_lang";
```

Comment:
```suggestion
    const ID_LANG = 'id_lang';
```
Single commas, please